javascript/ql/src/Security/CWE-079/UnsafeHtmlConstruction.qhelp

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+<overview>
+	<p>
+		Dynamically constructing HTML with inputs from library functions that are 
+		available to external clients may inadvertently leave a client open to XSS attacks.
+		
+		Clients using the exported function may use inputs containing unsafe HTML,
+		and if these inputs end up in the DOM, the client may be vulnerable to
+		cross-site scripting attacks.
+	</p>
+
+</overview>
+<recommendation>
+
+	<p>
+		If possible, use safe APIs when inserting HTML into the DOM.  
```

Comment:
```suggestion
		If possible, use safe APIs when modifying the DOM.  
```
When the suggestion below is to use `innerText`, it is confusion to see "when inserting HTML".